docker/build_scripts/config_radixdlt.sh

```diff
@@ -1,20 +1,6 @@
 #!/bin/bash
-
 set -ex
 
-# Need this to run on Alpine (grsec) kernels without crashing
-find /usr -type f -name java -exec setfattr -n user.pax.flags -v em {} \;
-
-# Sets USER_ID to LOCAL_USER_ID if provided, else set it to 999
```

Comment:
The lines below from down here should be kept intact. Tehy allow us to run the container as radixdlt user

Reply:
Those steps needed at runtime as the vars are only avaliable on run time and they vary on individual node runners.

Reply:
@shambupujar so I only wanted to move the user creation to the Dockerfile, would you be ok with keeping the user and group creation there?
As of env vars, would you please let me know which ones are needed? I've tested the docker image created and it is starting just fine

Reply:
resolving as per the agreement on slack